tests/irv/test_cases/silly_names.csv

```diff
@@ -0,0 +1,5 @@
+# winner:None
+None,null,-1,ü
```

Comment:
this opens up a very large can of worms haha. what do we have to support? numbers in names? non-latin characters? emojis? zero-width spaces? carriage returns? the possibilities are endless.